tutorecommerce/templates/ecommerce/build/ecommerce/Dockerfile

```diff
@@ -79,6 +80,12 @@ RUN cd /openedx/requirements/ \
 {% for extra_requirement in ECOMMERCE_EXTRA_PIP_REQUIREMENTS %}RUN --mount=type=cache,target=/openedx/.cache/pip,sharing=shared,uid=${APP_USER_ID} pip install '{{ extra_requirement }}'
 {% endfor %}
 
+{% if ECOMMERCE_ATLAS_PULL %}
+# Pull translations. Support the OEP-58 proposal behind a feature flag until it's fully implemented.
+RUN atlas pull {{ patch("atlas-extra-args") }} translations/ecommerce/ecommerce/conf/locale:ecommerce/conf/locale
```

Comment:
Please use the new format as seen in https://github.com/overhangio/tutor-credentials/pull/29/files#diff-1d0fcae46139cd49acb181fdd0a46cae537e257b78120b14bdc283b2db30054aR128